app/assets/stylesheets/govuk_publishing_components/components/_warning-text.scss

```diff
@@ -10,7 +14,7 @@
   margin-left: 0;
 }
 
-.gem-c-warning-text__text--large {
+.gem-c-warning-text .gem-c-warning-text__text--large {
```

Comment:
This looks like the specificity of the selector has been increased - is this for a CSS reason or just to match the rest of the file? Just checking if this is needed since these are `gem-c` classes.

Reply:
Yes, the main reason for this is that the font-size is now set for `.gem-c-warning-text .govuk-warning-text__text` - https://github.com/alphagov/govuk_publishing_components/pull/4041/files#diff-4d9503e344b5466e77c03df22484e4e39d15eb4144003d9a8ed763eb6561d49bR5-R8, so I've increased the specificity here to ensure is has a font-size of 24px instead of 19px.

Reply:
Great, thanks for the explanation 👍